clock_exercise/main.qml

```diff
@@ -42,10 +42,9 @@ Window {
         source: "fonts/BuenosAires_Family/BuenosAires-Book.otf"
     }
 
-
     StackView {
         id: stack
-        initialItem: clockPage
+        initialItem: dateSettingPage //clockPage
```

Comment:
Questo lo sto vedendo nel primo commit della PR: so che lo hai risolto successivamente, cerca comunque di fare attenzione quando vai a committare modifiche, in generale è preferibile non avere cambiamenti temporanei di questo tipo nel codice (sia QML che C++).
Siccome anche io spesso lavoro con modifiche di questo tipo (per una svariata serie di motivi), la cosa migliore che posso consigliarti per evitare questi errori è usare `git cola` per selezionare le singole righe da committare.

Se dovessi accorgerti a posteriori di aver ormai committatto qualcosa di estraneo dentro un commit, posso farti vedere come applicare correzioni a posteriori, chiaramente da usare in extremis ;)